libcxx/docs/Status/Cxx23Papers.csv

```diff
@@ -39,7 +39,7 @@
 "`P2401R0 <https://wg21.link/P2401R0>`__","Add a conditional ``noexcept`` specification to ``std::exchange``","2021-10 (Virtual)","|Complete|","14.0",""
 "","","","","",""
 "`P0323R12 <https://wg21.link/P0323R12>`__","``std::expected``","2022-02 (Virtual)","|Complete|","16.0",""
-"`P0533R9 <https://wg21.link/P0533R9>`__","``constexpr`` for ``<cmath>`` and ``<cstdlib>``","2022-02 (Virtual)","|In Progress|","","``isfinite``, ``isinf``, ``isnan`` and ``isnormal`` are implemented"
+"`P0533R9 <https://wg21.link/P0533R9>`__","``constexpr`` for ``<cmath>`` and ``<cstdlib>``","2022-02 (Virtual)","|In Progress|","","``isfinite``, ``isinf``, ``isnan``, ``isnormal`` and ``signbit`` are implemented"
```

Comment:
Instead, can we mark this as "in progress" and track what has been done in the Github issue for P0533R9? (https://github.com/llvm/llvm-project/issues/105174)

Reply:
Sure, will do so. Thanks for hint. I will revert this change in the CSV status files then. Tracking subtasks and encoding of custom notes via GitHub issues from now on 👍